### DISCUSSION TITLE
CARTO V1 with styles

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#6482] [Q&A]
Hello,

Is it possible to render CARTO layers also with texture created in CARTO?

It returns only .mvt tiles. I have some maps build in CARTO and I want to render it with styles.

Thanks

**Comment by AdriSolid** [comment]
- CARTO 2 (classic CARTO): unfortunately that's **not possible**. You would need to customize your `CartoLayer` styles, see [docs](https://docs.carto.com/deck-gl/customizing-the-cartolayer-style/).

- CARTO 3: you will be able to achieve that creating a map-id from your map. deck.gl supports that already, not sure if docs are ready, @borja-munoz any idea?

**Reply by borja-munoz** [reply]
Yes, docs in the master branch have already been updated with information about the new `fetchMap` function: https://github.com/visgl/deck.gl/blob/master/docs/api-reference/carto/fetch-map.md
